app/src/main/res/layout/offer_peril_item.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.hedvig.app.ui.view.HedvigCardView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content">
+
+    <LinearLayout xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="@dimen/offer_peril_card_width"
```

Comment:
I think it should be possible to allow the width and height of this card to be entirely derived. Using `layout_width="match_parent"` and `layout_height="wrap_content"` and setting up the margins to be as specified in the designs should give us the correct result, scaling in a satisfying manner to different screen sizes 🙂

Reply:
This is still not fixed 🤔